plugins/callback/loganalytics.py

```diff
@@ -0,0 +1,252 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: loganalytics
+    type: aggregate
+    short_description: Posts task results to Azure Log Analytics
+    author: "Cyrus Li <cyrus1006@gmail.com>"
```

Comment:
```suggestion
    author: "Cyrus Li (@zhcli) <cyrus1006@gmail.com>"
```
Then the bot knows whom to ping if there are issues or PRs for this plugin :)